src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall(){
+        Ball ball = new Ball();
+        ball.setColor(new ColorSupplier().getRandomColor());
+        Random random = new Random(100);
```

Comment:
The `Random` constructor should not take an argument. Instead, use `random.nextInt(100)` to generate a random number between 0 and 99, or specify another range if needed.